src/Microsoft.AspNetCore.Mvc.Razor/Infrastructure/IFileVersionProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Infrastructure
```

Comment:
Consider just `Mvc.Infrastructure` for this. It's not really tied to Razor in any way